koans/AboutApplyingWhatWeHaveLearnt.js

```diff
@@ -55,14 +57,16 @@ describe("About Applying What We Have Learnt", function() {
       }
     }
 
-    expect(sum).toBe(FILL_ME_IN);
+    expect(sum).toBe(233168);
   });
 
   it("should add all the natural numbers below 1000 that are multiples of 3 or 5 (functional)", function () {
 
-    var sum = FILL_ME_IN;    /* try chaining range() and reduce() */
+    var sum =
```

Comment:
This should use `_.range` and then using `.reduce` to sum up the different numbers from 0 to 1000